source/supported-operations.txt

```diff
@@ -368,16 +382,42 @@ Information Functions
    * - ``DATABASE()``
      - Return the default (current) database name
    * - ``SCHEMA()``
-     - Synonym for DATABASE()
+     - Synonym for ``DATABASE()``
    * - ``SESSION_USER()``
-     - Synonym for USER()
+     - Synonym for ``USER()``
    * - ``SYSTEM_USER()``
-     - Synonym for USER()
+     - Synonym for ``USER()``
    * - ``USER()``
      - The user name and host name provided by the client
    * - ``VERSION()``
      - Return a string that indicates the MySQL server version
 
+Utility Functions
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Function
+     - Description
+
+   * - ``USE``
```

Comment:
For the utility functions section, would it be worthwhile to give the usage template? e.g. instead of just ` ``USE`` `, have ` ``USE <db>`` `